packages/opentelemetry-core/src/index.ts

```diff
@@ -29,5 +29,6 @@ export * from './trace/NoRecordingSpan';
 export * from './trace/sampler/ProbabilitySampler';
 export * from './trace/spancontext-utils';
 export * from './trace/TraceState';
+export * from './utils/trace-flags';
```

Comment:
I would not export `trace-flags.ts`, looks like this is used for internal operation only.